core/modules/admin_bar/css/admin_bar.css

```diff
@@ -83,6 +83,16 @@
   padding-right: 28px;
 }
 
+/* Back to Site arrow. */
+
+#admin-bar .dropdown li.admin-bar-icon > a.escape {
+  background-image: url("../icons/chevron-disc-left.png");
```

Comment:
The new icons (added below) don't look good on high resolution displays. To make them look better, the images should be 32 x 32px instead of 16px.

Reply:
I've also wondered if it is better to place these in the images folder with everything else rather than a separate icons folder.  The icons came from the Drupal module [](https://www.drupal.org/project/escape_admin).

Reply:
> if it is better to place these in the images folder with everything else rather than a separate icons folder.

Agreed!